packages/postgres-database/src/simcore_postgres_database/models/license_goods.py

```diff
@@ -0,0 +1,67 @@
+""" resource_tracker_service_runs table
+"""
+
+import enum
+
+import shortuuid
+import sqlalchemy as sa
+
+from ._common import RefActions, column_created_datetime, column_modified_datetime
+from .base import metadata
+
+
+def _custom_id_generator():
+    return f"lgo_{shortuuid.uuid()}"
+
+
+class LicenseResourceType(str, enum.Enum):
+    VIP_MODEL = "VIP_MODEL"
+
+
+license_goods = sa.Table(
+    "license_goods",
+    metadata,
+    sa.Column(
+        "license_good_id",
+        sa.String,
```

Comment:
question, why is it a short UUID, and also why we do not start using sa.UUID ?

Reply:
look a bit above, there is:

```
def _custom_id_generator():
    return f"lgo_{shortuuid.uuid()}"
```

The inspiration was taken from Stripe resources and how they handle unique identifiers (which is pretty useful during debugging). They use a small prefix, usually starting with the resource name (e.g., lgo for "license goods"), followed by a short UUID without dashes (-), which provides sufficient randomness. Additionally, we want to move away from integers to avoid issues with migrations.

This is just an idea to see whether you like it. If we decide to use UUIDs, I’m fine with that as well. Once again, please share your opinions, and I will make the changes accordingly. Thanks!

Reply:
I personally like this approach and added it as a comment in https://github.com/ITISFoundation/osparc-simcore/issues/6870

Reply:
ACtually regarding UUIDs, and I also already talked with @pcrespov about it.
I think that prj_UUID is ok for outside the database. Inside the database in the projects table I don't see why we would put such a prefix. What does it bring there? nothing. It just forces us to use a STRING instead of a UUID type.

Chatty says that:
- using a UUID type is better for storage efficency
- better for performance
- better for data integrity
- using a STRING is more flexible because you can put whatever...

Reply:
I still do not have a strong opinion, so my response is more to challenge the 'nothing' statement. For sure, the advantage of having it inside the database is that it will be much less error-prone when you have relationships through these IDs. I think it will cause us far fewer troubles than always having to think about where this "prefix" should be added on the fly.

Reply:
Regarding storage efficiency, this can also be addressed if we have a fixed length of string, for example, a prefix with 3 characters, an underscore, and a short UUID.

Reply:
But if you agree, for now, I will just use a pure UUID with the UUID type. This should be sufficient, and if we want to add the prefix in the future, we can easily perform a migration.

Reply:
Conclusion:
![image](https://github.com/user-attachments/assets/71ffbdd4-5aba-4187-a880-be5bf40dd55f)

Reply:
nice. I still think the prefix is only useful if you send stuff outside. or if you export stuff. Anyway I like your solution.